internal/performance/cleanup_test.go

```diff
@@ -0,0 +1,226 @@
+// +build performance
```

Comment:
This looks _very_ similar to the existing integration test. Is there a way we can merge them into a single test to reduce duplication?

Reply:
What is "reasonable time"? Are we talking hours or minutes? I think it's okay if the integration tests take a few minutes to be honest. Especially if we can reduce the maintenance overhead.

Reply:
I'm afraid it's hours, the exports has to be done in sequential order, and considering the db lock, each request takes at least 2 seconds in my past experiments, so we are talking about 2 * 10000 seconds, which is almost 6 hours of time here. 

I do agree with you that it's desired to reduce this the maintenance overhead, will spend some more time to see whether I can squeeze the time spent for each export

Reply:
Ack, issue created here https://github.com/google/exposure-notifications-server/issues/751, and the tests were fixed